doc/src/main/asciidoc/inc/_enricher.adoc

```diff
@@ -967,8 +975,8 @@ The following configuration parameters can be used to override the behavior of t
 | The version of the 3scale discovery implementation.
 | `v1`
 
-| *discoverable*
-| Sets the discoverable label to either true or false. If it's set to "false" 3scale will not try to discover this service.
+| *discovery.3scale.net*
```

Comment:
sorry, again me: The configuration is called 'discoverable', isn't it ? It's only the label which is different.

Reply:
Yes! Good catch I search/replaced too aggressively.